### ISSUE TITLE
Measure distance ruler disappears when using JC hot key

### ISSUE THREAD

**Issue by terracoda** [phetsims/coulombs-law#128]
The ruler in this sim should work the same as the ruler in Gravity Force Lab (regular) which is accessible with keyboard and on iOS VoiceOver.

Currently, there are some bugs with keyboard interaction on macOs in Safari.

The ruler disappeared from view when I used the JC hot key to align the start of the ruler with the center of the q1 charged sphere.

I have not tested on mobile.

Designers, should also check pdom order of the measure distance ruler. In GFL, using the ruler is optional so it is placed further down the pdom order. This order may also be appropriate for the ruler in this sim.

**Comment by terracoda** [comment]
Noting these issues in case there is an upcoming maintenance release.

**Comment by jessegreenberg** [comment]
I just noticed this while working on https://github.com/phetsims/scenery/issues/1354. I tested gravity-force-lab and it is not an issue there even though the hotkey is implemented in inverse-square-law-common.

**Comment by terracoda** [comment]
A lot of work has gone into alternative input since CL was published with that feature. It might be missing some upgrades that GFL already has - maybe even just in the HTML code and the HTML and ARIA attribute. Like I am sure CL's rule does not have an accessible name through `aria-roledescription`, and the web application role might be implemented differently.

**Comment by jessegreenberg** [comment]
Just ran into this again, still a problem.
